packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -68,11 +68,11 @@ const EventHandlerTrackerAttribute = `data-interactable`;
 
 const InteractableAncestor = `interactableAncestor`;
 type InteractableAncestorCache = {
-  [index: string]: (HTMLElement | null)[];
+  [index: string]: (HTMLElement | SVGElement | null)[];
```

Comment:
In all of these cases, I think we can simply convert them to `Element`. The api we need from the object should be in Element

https://developer.mozilla.org/en-US/docs/Web/API/Element

This may also make some of the check unnecessary, since some other browser API may return `Element` and usually we should check their type.

Reply:
Element seems to work,  I wasn't sure if we were using some specific HTMLElement property not available on Element but it seems TS does not complain and it seems to work as expected.